lib/unit-ruby/util/resource_operations.rb

```diff
@@ -11,6 +11,21 @@ def find(id)
 
           build_resource_from_json_api(located_resource)
         end
+
+        def find_all(ids)
```

Comment:
Could we use the List operation here instead?

Reply:
sorry, do you mean update the existing `def list` so that it's able to determine whether the resource is a single resource or a plural resource?

Reply:
Also I realized that my original implementation could be simplified, what's currently reflected is the more streamlined version.